infrateam_first_responder.md

```diff
@@ -54,6 +54,10 @@ If on a Mac, you will get better results if you stay in the same "space" as the
 
 To complete the cycle, and ensure QA has the common environment, deploy there as well. Then, **warn #dlss-infra-stage-qa-use** of the impending deployment to QA in case there is active testing going on; if so, be sure to either skip that app or coordinate with tester and then deploy the tag you created above to QA using `sdr-deploy`.
 
+##### 6. Deploy to AWS
+
+The [speech-to-text](https://github.com/sul-dlss/speech-to-text) project is deployed automatically to qa and stage environments in AWS when a version is tagged `release-YYYY-MM-DD`. If the integration tests runs clean then you can trigger a production deploy by manually [creating a release](https://github.com/sul-dlss/speech-to-text/releases/new) in Github, using the `release-YYYY-MM-DD` tag.
+
```

Comment:
I believe the pattern we tend to use for weekly update tags is `rel-{YYYY}-{MM}-{DD}`.

```suggestion
The [speech-to-text](https://github.com/sul-dlss/speech-to-text) project is deployed automatically to qa and stage environments in AWS when a version is tagged `rel-YYYY-MM-DD`. If the integration tests runs clean then you can trigger a production deploy by manually [creating a release](https://github.com/sul-dlss/speech-to-text/releases/new) in Github, using the `rel-YYYY-MM-DD` tag.

```

Reply:
Yes, thanks for catching that! That's the way the Github Action works currently too.